KarhooUISDK/Screens/CheckoutScreen/NewCheckout/Workers/KarhooNewCheckoutLoyaltyWorker.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  KarhooNewCheckoutLoyaltyWorker.swift
+//  KarhooUISDK
+//
+//  Created by Aleksander Wedrychowski on 17/01/2023.
+//  Copyright © 2023 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import KarhooSDK
+
+protocol NewCheckoutLoyaltyWorker: AnyObject {
```

Comment:
@AleksanderWedrychowskiKarhoo sync with my branch. You have a loyalty pre-auth worker there. I've broken the initial presenter's logic/

Reply:
I will adjust it after both branches are merged. Access point is almost excacly the same, so it's not a big deal.